apf/apf.h

```diff
@@ -704,6 +706,9 @@ bool isPrintable(Field* f);
   */
 void fail(const char* why) __attribute__((noreturn));
 
+
+
+
```

Comment:
why is there all this extra whitespace?

Reply:
just crept into the commit, I've fixed this and once I squash the commits it will not be in the merge